docs/products/networking/cloud-firewall/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Cloud Firewall
+description: "Linode Cloud Firewall offers a simple interface to protect your web apps. It is scalable security in seconds, allowing you to create custom firewall rules, making security more accessible."
+tab_group_main:
+    is_root: true
+    title: Overview
+    weight: 10
+cascade:
+    date: 2020-06-02
+    product_description: "Secure your network with Linode Cloud Firewall. Filter traffic by allowing it on selected ports and protocols."
```

Comment:
Copy edit suggestion for clarity: "Secure your network with Linode Cloud Firewall. Create Firewall rules that only allow network traffic on selected ports and protocols."

Reply:
Can we suggest this for the marketing page as well - this was taken directly from the marketing copy: https://www.linode.com/products/cloud-firewall/